lib/utils/type-tracker/index.ts

```diff
@@ -587,6 +592,7 @@ export function createTypeTracker(context: Rule.RuleContext): TypeTracker {
         const declarations = symbol && symbol.declarations
         const declaration = declarations && declarations[0]
         if (
+            declaration &&
```

Comment:
This doesn't look like reformating. There are also a few other changes of this sort in `lib/utils/type-tracker/**` files. What's that about?

Reply:
Ok, that's fine.